cerebrium/logging.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "External Monitoring Tools"
+description: "Logging your predictions to external monitoring tools."
+---
+
+# External Monitoring Tools
```

Comment:
I would create a grouping structure like I did for the prebuilt models so folder structure would be:
- Logging
    - Censius
    - Arize
    - etc

The logging docs for each platform would be pretty long since you should attached screenshots and step by steps to do it